comparison.md

```diff
@@ -109,19 +106,19 @@ AssertJ provides assertions for several libraries' types. As of this writing,
 Truth includes assertions for [Guava] and [Protocol Buffers].
 
 Both have third-party extensions, such as for Android types
-([AssertJ][AssertJ-Android], [Truth][Truth-Android]). I don't have a feel for
-the overall size of each ecosystem.
+([AssertJ][AssertJ-Android], [Truth][Truth-Android]).
+It's unclear how large each ecosystem is overall.
```

Comment:
This may also have changed since when Chris first published this.

Reply:
I still haven't tried to measure it. Maybe we'll do so someday.